staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go

```diff
@@ -43,13 +43,13 @@ type Aggregate interface {
 // this returns nil.
 // It will check if any of the element of input error list is nil, to avoid
 // nil pointer panic when call Error().
-func NewAggregate(errlist []error) Aggregate {
-	if len(errlist) == 0 {
```

Comment:
changing variable names like this masks history for little benefit… the existing variable names in this PR are generally fine as they were. Let's limit this PR to correcting actual typos